uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeaveRequest.php

```diff
@@ -336,8 +336,9 @@ private static function validateTOILPastDays($params, $absenceType) {
     $toDate = new DateTime($params['to_date']);
     $todayDate = new DateTime('today');
     $leaveDatesHasPastDates = $fromDate < $todayDate || $toDate < $todayDate;
+    $isACancellationRequest = in_array($params['status_id'], LeaveRequest::getCancelledStatuses());
```

Comment:
If you are updating a leave request without changing the status will this mean `status_id` won't be set? If it isn't then it would trigger a warning

Reply:
The status_id is always required whether creating or updating a leave request.

Reply:
Hmm, not exactly standard behavior for the API, but if that's the case then you can ignore my comment

Reply:
I agree. The Leave Request API is an exception.